components/Typography.jsx

```diff
@@ -64,3 +64,33 @@ export const Code = styled("code", {
   borderWidth: 1,
   borderRadius: "$3",
 });
+
+export const IconButton = styled("button", {
```

Comment:
- I'd prefer if we inverted the colors, so it has an `$orange` outline and fill on hover.
- Please reduce the scaling to 2-3%.
- This component should live in a new file `Buttons.jsx` since this file is just for type.
- Add `fontFamily: "$mono", fontSize: "$2"`